packages/grid/data-grid/src/tests/selection.DataGrid.test.tsx

```diff
@@ -12,6 +12,49 @@ describe('<DataGrid /> - Selection', () => {
   // TODO v5: replace with createClientRender
   const render = createClientRenderStrictMode();
 
+  it('should be selected and deselected when clicking on a row', () => {
```

Comment:
If we split, it will be easier to understand what should happen in this test and in the selection behavior, like holding CTRL to unselect, or selecting multiple rows...